supportedBackends/google/pipelines/v1alpha2/src/main/scala/cromwell/backend/google/pipelines/v1alpha2/api/request/RequestHandler.scala

```diff
@@ -1,35 +1,47 @@
 package cromwell.backend.google.pipelines.v1alpha2.api.request
 
-import java.net.URL
-
 import akka.actor.ActorRef
 import com.google.api.client.googleapis.batch.BatchRequest
 import com.google.api.client.googleapis.json.GoogleJsonError
-import com.google.api.client.http.HttpRequest
 import com.google.api.services.genomics.Genomics
+import cromwell.backend.google.pipelines.common.PipelinesApiConfigurationAttributes.BatchRequestTimeoutConfiguration
 import cromwell.backend.google.pipelines.common.api.PipelinesApiRequestHandler
 import cromwell.backend.google.pipelines.common.api.PipelinesApiRequestManager.{PAPIAbortRequest, PAPIApiRequest, PAPIRunCreationRequest, PAPIStatusPollRequest}
 import cromwell.cloudsupport.gcp.auth.GoogleAuthMode
 
+import java.net.URL
 import scala.collection.JavaConverters._
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
 
-class RequestHandler(applicationName: String, endpointUrl: URL) extends PipelinesApiRequestHandler 
+class RequestHandler(applicationName: String,
+                     endpointUrl: URL,
+                     batchRequestTimeoutConfiguration: BatchRequestTimeoutConfiguration
+                    )
+  extends PipelinesApiRequestHandler
   with RunRequestHandler
   with GetRequestHandler 
   with AbortRequestHandler {
 
-  override def makeBatchRequest = new Genomics.Builder(GoogleAuthMode.httpTransport, GoogleAuthMode.jsonFactory, (_: HttpRequest) => ())
-    .setApplicationName(applicationName)
-    .setRootUrl(endpointUrl.toString)
-    .build().batch()
+  override def makeBatchRequest: BatchRequest = {
+    val builder =
+      new Genomics.Builder(
+        GoogleAuthMode.httpTransport,
+        GoogleAuthMode.jsonFactory,
+        initializeHttpRequest(batchRequestTimeoutConfiguration) _,
```

Comment:
TOL: If `batch()` ignores the original builder's `httpRequestInitializer`, do we have to specify it here or can we just do this below?

```
client.batch(initializeHttpRequest(batchRequestTimeoutConfiguration) _)
```

I'm only just thinking of it now, but I think this would apply in other places in this PR as well.

Reply:
AFAICT you do NOT need to specify it in both places. But some code (`v2Beta`, `v2Alpha1` but not `v1Alpha2`) were specifying the initializer in _both_ the single and batch requests already, so I just spread that "use same init in both single and batch" style around.